Userland/Libraries/LibVideo/VP9/Parser.h

```diff
@@ -119,16 +121,25 @@ class Parser {
     DecoderErrorOr<i32> read_coef(Token token);
 
     /* (6.5) Motion Vector Prediction */
-    DecoderErrorOr<void> find_mv_refs(ReferenceFrame, int block);
-    DecoderErrorOr<void> find_best_ref_mvs(int ref_list);
-    DecoderErrorOr<void> append_sub8x8_mvs(u8 block, u8 ref_list);
-    DecoderErrorOr<bool> use_mv_hp(MotionVector const& delta_mv);
+    void find_mv_refs(ReferenceFrame, i32 block);
+    void find_best_ref_mvs(u8 ref_list);
+    bool use_mv_hp(MotionVector const& delta_mv);
+    void append_sub8x8_mvs(i32 block, u8 ref_list);
+    bool is_inside(i32 row, i32 column);
+    void clamp_mv_ref(u8 i);
+    MotionVector clamp_mv(MotionVector mvec, i32 border);
     size_t get_image_index(u32 row, u32 column);
+    void get_block_mv(u32 candidate_row, u32 candidate_column, u8 ref_list, bool use_prev);
+    void if_same_ref_frame_add_mv(u32 candidate_row, u32 candidate_column, ReferenceFrame ref_frame, bool use_prev);
+    void if_diff_ref_frame_add_mv(u32 candidate_row, u32 candidate_column, ReferenceFrame ref_frame, bool use_prev);
```

Comment:
these two function names feel out of style

It's a bit sad that the old function names were using so many abbreviations. would be nice to replace all the `mv` with `motion_vector` at some point (after this PR)

But for these two in particular, they should be something more like
`add_mv_if_ref_frame_same`

to keep the verb towards the front of the method name.

Reply:
For all the initial implementations, I was using the exact naming from the spec (converting to snake case or camel case where needed), what are your thoughts on leaving this for a later refactor?

I had planned on going through and cleaning up the decoder to make it more readable and fix up unnecessarily stored state once it was fully functional, to avoid uprooting the existing implementation and breaking things.

Reply:
yah this is fine for a later refactor.